woocommerce/woocommerce-gutenberg-products-block
# Shopper → Mini Cart → Can go to Cart page

Fixes #5836

### Notes

This PR adds e2e tests to verify that a shopper can go to the Cart page from the Mini Cart drawer.

### Testing

#### Automated Tests

### Manual Testing

1. Check out this PR.
2. Start Docker Desktop.
3. Run `npm run wp-env destroy && npm run wp-env start` to start a fresh e2e environment.
4. Run `npm run test:e2e -- tests/e2e/specs/shopper/mini-cart.test.js -t "Cart page"` to run only this test case.
5. Run `npm run test:e2e` to run the whole test suite.
